.github/workflows/update-gatewayrs.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
```

Comment:
i think repository_dispatch or workflow_dispatch actions have to be in the master branch to work

Reply:
![image](https://user-images.githubusercontent.com/42802937/160998770-2bc69918-873c-4bc4-a820-2f0f5f72f4d3.png)

Checked. We can run on any branch.

Reply:
You can *run* it on any branch, but the workflow has to be stored in master

Reply:
On further investigation, it's just the repository_dispatch that works in this way. Workflow dispatch seems to be ok...
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch

But for repository dispatch:
> Note: This event will only trigger a workflow run if the workflow file is on the default branch.

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#repository_dispatch

Reply:
So, truce then? :slightly_smiling_face: